Formula/x/x86_64-elf-grub.rb

```diff
@@ -0,0 +1,65 @@
+class X8664ElfGrub < Formula
+  desc "GNU GRUB bootloader for x86_64-elf"
+  homepage "https://savannah.gnu.org/projects/grub"
+  url "https://ftp.gnu.org/gnu/grub/grub-2.06.tar.xz"
+  mirror "https://mirrors.ocf.berkeley.edu/gnu/grub/grub-2.06.tar.xz"
+  sha256 "b79ea44af91b93d17cd3fe80bdae6ed43770678a9a5ae192ccea803ebb657ee1"
+  license "GPL-3.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gcc@11" => [:build, :test]
+  depends_on "help2man" => :build
+  depends_on "make" => :build
+  depends_on "objconv" => :build
+  depends_on "texinfo" => :build
+  depends_on "x86_64-elf-binutils" => :build
+  depends_on "x86_64-elf-gcc" => [:build, :test]
+  depends_on "gettext"
+  depends_on "xorriso"
+  depends_on "xz"
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
+  uses_from_macos "python" => :build
+
+  def install
+    target = "x86_64-elf"
+    ENV["CFLAGS"] = "-Os -Wno-error=incompatible-pointer-types"
+
+    mkdir "build" do
+      args = %W[
+        --disable-werror
+        --target=#{target}
+        --prefix=#{prefix}/#{target}
+        --bindir=#{bin}
+        --libdir=#{lib}/#{target}
+        --with-platform=pc
+        --program-prefix=#{target}-
+        TARGET_CC=#{target}-gcc
```

Comment:
I'm surprised because GNU tools are usually able to detect these target tools if the correct `-build`, `-host` and `-target` arguments (one or more of them, probably `-target` in this case) is provided.